controllers/operands/ssp_test.go

```diff
@@ -756,14 +756,37 @@ var _ = Describe("SSP Operands", func() {
 					goldenImageList, err := getDataImportCronTemplates(hco)
 					Expect(err).ToNot(HaveOccurred())
 					Expect(goldenImageList).To(HaveLen(3))
-					Expect(goldenImageList).To(HaveCap(4))
 
 					statusImage2Enabled := statusImage2.DeepCopy()
+					enableDict(&statusImage2Enabled.DataImportCronTemplate)
 					statusImage2Enabled.Status.Modified = true
 
 					Expect(goldenImageList).To(ContainElements(*statusImage2Enabled, statusImage3, statusImage4))
 				})
 
+				It("Should not add a common DIC template if it is disabled in the file", func() {
+					disabledImage1 := image1.DeepCopy()
+					disableDict(disabledImage1)
+					enabledImage2 := image2.DeepCopy()
+					enableDict(enabledImage2)
+
+					dataImportCronTemplateHardCodedMap = map[string]hcov1beta1.DataImportCronTemplate{
+						image1.Name: *disabledImage1,
+						image2.Name: *enabledImage2,
+					}
+					hco := commontestutils.NewHco()
+					hco.Spec.FeatureGates.EnableCommonBootImageImport = ptr.To(true)
+
+					goldenImageList, err := getDataImportCronTemplates(hco)
+					Expect(err).ToNot(HaveOccurred())
+					Expect(goldenImageList).To(HaveLen(1))
+
+					statusImage2Enabled := statusImage2.DeepCopy()
+					enableDict(&statusImage2Enabled.DataImportCronTemplate)
+
+					Expect(goldenImageList).To(ContainElements(*statusImage2Enabled))
+				})
+
```

Comment:
I think we need to test different scenario instead of this: allow disabling a user defined DICT.